samples/unit-tests/gantt/xrange-series/demo.js

```diff
@@ -255,7 +259,14 @@
         assert.notEqual(
             chart.tooltip.label.element.textContent.indexOf('Monday'),
             -1,
-            'The tooltip header should container formatted date (#9301)'
+            'The tooltip header should contain a formatted date (#9301)'
+        );
+
+        chart.series[0].update({ color: 'red' });
+        assert.strictEqual(
+            chart.series[0].color,
+            'red',
+            'The series color should have changed to red'
```

Comment:
If I comment out your fix from the HC code (so the code will be as on the master branch), then the test has passed (success), but the point is green, not red. The test is not catching the issue.

Reply:
Alright, should work now, testing `point.color` instead of `series.color`, which is green without the fix